test/governance/TimelockController.test.js

```diff
@@ -632,6 +709,84 @@ contract('TimelockController', function (accounts) {
                 'TimelockController: length mismatch',
               );
             });
+
+            it('prevents reentrancy execution', async function () {
+              // Create operation
+              const reentrant = await TimelockReentrant.new();
+              const reentrantBatchOperation = genOperationBatch(
+                Array(8).fill(reentrant.address),
+                Array(8).fill(0),
+                Array(8).fill(reentrant.contract.methods.reenter().encodeABI()),
```

Comment:
I think I would use a batch of 1, it's simpler and seems more "likely" to be vulnerable to reentrancy.

```suggestion
                [reentrant.address],
                [0],
                [reentrant.contract.methods.reenter().encodeABI()],
```